analyzers/src/SonarAnalyzer.CSharp/SymbolicExecution/Sonar/SonarExplodedGraph.cs

```diff
@@ -576,6 +584,13 @@ protected override void VisitInstruction(ExplodedGraphNode node)
                     // Do nothing
                     break;
 
+                case SyntaxKindEx.ParenthesizedPattern:
```

Comment:
See the cases above for why I added the new cases.

Reply:
Actually here, I don't see any reason why we would want to throw the `NotSupportedException` anymore.

Let's just `// Do nothing` in the `default` clause. As this whole thing is deprecated. And it only causes trouble that we don't want to solve (like this PR :) )

Reply:
Here I'd still like tor emove the throwing part, as there's no reason for it in the future